charts/sonarqube-dce/templates/_helpers.tpl

```diff
@@ -229,3 +229,29 @@ set search.ksPassword
 {{- template "sonarqube.fullname" . -}}-keystore-pass
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return the target Kubernetes version
+*/}}
+{{- define "common.capabilities.kubeVersion" -}}
+{{- if .Values.global }}
```

Comment:
How are `Values.global` parameters set? Is this something set by helm while outputting a template? In any case, we should quickly document/comment it as they are not listed among our parameters :)

Reply:
If those new values are only meant for testing, I think it is preferable not to document them as this is not for public use

Reply:
Global is Helm specific, it allows to pass value in subChart/parentChart with the same root key ```Global```

At first i thought it might be great for user to have a way of specifying what kube_version they want to target, so that if the cluster is upgraded without them knowing the chart stay the same.

But after thinking about it, it can only introduce compatibility issue, so i would suggest to remove everything and keep only 
```.Capabilities.KubeVersion.Version ``` which is either retrieve from the cluster or from --kube-version on the helm template side.

What do you guys think ?

Reply:
If we can easily change the Kube  version between tests with that, I think this is the preferable solution to not add additional complexity to the chart